README.md

```diff
@@ -80,13 +80,13 @@ Please, do not open issues for the general support questions as we want to keep
 
 StackOverflow is a much better place to ask questions since:
 * there are hundreds of people willing to help on StackOverflow
-* questions and answers stay available for public viewing so your question / answer might help someone else
-* SO voting system assures that the best answers are prominently visible.
+* questions and answers stay available for public viewing so your question/answer might help someone else
+* SO the voting system assures that the best answers are prominently visible.
```

Comment:
In this case SO means Stack Overflow, so please either:
- rollback or
- replace SO with Stack Overflow's